components/sup/src/manager/commands.rs

```diff
@@ -185,60 +184,39 @@ pub fn service_load(mgr: &ManagerState,
                     opts: &protocol::ctl::SvcLoad)
                     -> NetResult<()> {
     let ident: PackageIdent = opts.ident.clone().ok_or_else(err_update_client)?.into();
-    let bldr_url = opts.bldr_url
-                       .clone()
-                       .unwrap_or_else(|| protocol::DEFAULT_BLDR_URL.to_string());
-    let bldr_channel = opts.bldr_channel
-                           .clone()
-                           .map(ChannelIdent::from)
-                           .unwrap_or_default();
-    let force = opts.force.unwrap_or(false);
     let source = InstallSource::Ident(ident.clone(), PackageTarget::active_target());
-    match mgr.cfg.spec_for_ident(source.as_ref()) {
-        None => {
-            let mut spec = ServiceSpec::default();
-            opts.into_spec(&mut spec);
-
-            // We don't have any record of this thing; let's set it up!
-            //
-            // If a package exists on disk that satisfies the
-            // desired package identifier, it will be used;
-            // otherwise, we'll install the latest suitable
-            // version from the specified Builder channel.
-            util::pkg::satisfy_or_install(req, &source, &bldr_url, &bldr_channel)?;
-
-            mgr.cfg.save_spec_for(&spec)?;
-            req.info(format!("The {} service was successfully loaded", spec.ident))?;
+    let mut spec = if let Some(spec) = mgr.cfg.spec_for_ident(source.as_ref()) {
+        let force = opts.force.unwrap_or(false);
+        // We've seen this service  before. Thus `load`
```

Comment:
[nit] Extra space in comment here.
```suggestion
        // We've seen this service before. Thus `load`
```